src/os/bluestore/BlueStore.cc

```diff
@@ -4395,6 +4411,31 @@ bool BlueStore::is_rotational()
   return rotational;
 }
 
+bool BlueStore::is_journal_rotational()
+{
```

Comment:
The bluefs impl willb e somethhing like
<pre>
if (!bdev[BDEV_WAL] || bdev[BDEV_WAL]->is_rotational()) return true;
return false;
</pre>